src/components/players/list/index.js

```diff
@@ -12,40 +13,35 @@ class List extends React.Component {
     constructor(props) {
         super(props);
 
-        this.ref = createRef();
-
         this.state = {
-            selectedId: null,
+            selectedPlayerId: null,
             selectedTeam: this.props.selectedTeam,
         };
     };
 
     componentDidUpdate(prevProps) {
```

Comment:
The scroll to top upon team change has stopped working, let's add it in componentDidUpdate based on `isLoading` state being true.